sushy/interop_profiles/ServiceRoot.v1_0_0.json

```diff
@@ -0,0 +1,73 @@
+{
+	"SchemaDefinition": "RedfishInteroperabilityProfile.v1_0_0",
+	"ProfileName": "RedfishProfile",
+	"ProfileVersion": "1.0.0",
+	"Purpose": "Redfish Interoperability profile",
+	"OwningEntity": "Dell",
+	"ContactInfo": "abc@dell.com",
```

Comment:
Please add a `Protocol` object which describes "Redfish protocol functionality that is not related to the supported schemas or properties".

Information about that object can be found in the Distributed Management Task Force (DMTF) [Redfish Interoperability Profile specification](https://www.dmtf.org/sites/default/files/standards/documents/DSP0272_1.3.0.pdf), section 8.3 "Protocol requirements", pp. 10-12. Also, several [Open Compute Project's interop profiles](https://github.com/opencomputeproject/OCP-Profiles) contain this object.

Reply:
Added Protocol section